src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -3,30 +3,61 @@
 import core.basesyntax.dao.CommentDao;
 import core.basesyntax.model.Comment;
 import java.util.List;
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
 
 public class CommentDaoImpl extends AbstractDao implements CommentDao {
     public CommentDaoImpl(SessionFactory sessionFactory) {
         super(sessionFactory);
     }
 
     @Override
-    public Comment create(Comment entity) {
-        return null;
+    public Comment create(Comment comment) {
+        Transaction transaction = null;
+        try (Session session = factory.openSession()) {
+            transaction = session.beginTransaction();
+            session.persist(comment);
+            transaction.commit();
+            return comment;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
```

Comment:
In this case, we need to use the block "try-catch- finally"  for this operations, (good explaining in the next lesson https://www.youtube.com/watch?v=nbs0v7yim0Q&feature=youtu.be&ab_channel=Mateacademy )

Reply:
Thanks a lot, I've implemented it.